python/tvm/meta_schedule/cost_model/cost_model.py

```diff
@@ -175,6 +175,7 @@ def update(
         context: TuneContext,
         candidates: List[MeasureCandidate],
         results: List[RunnerResult],
+        skip_model: bool = False,
```

Comment:
Instead let's add a parameter `frozen: bool` to subclasses (XGBModel, MLPModel) constructor, and then `self.frozen == True`, the `update` method will return immediately without retraining or incremental training

Reply:
Addressed for the MLP model. Let me address other models later :)